src/entities/applet/lib/types.ts

```diff
@@ -1,4 +1,8 @@
-import { AppletEncryptionDTO } from '@shared/api';
+import {
+  AnalyticsResponseType,
+  AppletEncryptionDTO,
+  ResponseConfig,
+} from '@shared/api';
```

Comment:
We should not rely on the backend data contracts. Please define all necessary types here, on entity level, and then create mapper(s) in order to map DTOs to our business logic's interfaces